### PR TITLE
Add a simple fairing job example by using private-registry on Jupyter-notebook. 

### PR DESCRIPTION
**Pull request by mojokb**
**What this PR does / why we need it**:
Users who use a private registry do not have an example to simply test fairing-job. So, I made an example that I can test with append builder on jupyter-notebook.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/fairing/419)

### CODE REVIEW

File: examples/notebook/hello-world-with-private-registry.ipynb [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      "WARNING:kubeflow.fairing.kubernetes.manager:Waiting for fairing-job-x8qqq-wwrf9 to start...\n"
+     ]
+    },
+    {
+     "ename": "KeyboardInterrupt",
+     "evalue": "",
+     "output_type": "error",
+     "traceback": [
+      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
```
Comment (jinchihe) [inline examples/notebook/hello-world-with-private-registry.ipynb]: There is traceback? do not interrupt or clear outputs?

**Comment by jinchihe** [comment]
ok-to-test

File: examples/notebook/hello-world-with-private-registry.ipynb (hunk 1 above)
Comment (mojokb) [reply examples/notebook/hello-world-with-private-registry.ipynb]: I don't know why "output" is included. I'll fix it.

**Comment by abhi-g** [comment]
@mojokb Please merge upstream/master to pull in some fixes

**Comment by jinchihe** [comment]
I think that's caused by testing env problems...
```
Step #2: E           RuntimeError: Remote file gs://kubeflow-ci-fairing/lightgbm/example/model_2019_11_22_06_09_21.txt does't exist
Step #2:
Step #2: /usr/local/lib/python3.7/site-packages/papermill/execute.py:192: PapermillExecutionError
```

/test kubeflow-fairing-presubmit

**Pull request merged**